File: src/main/java/br/com/guizzo/projectbasic/modules/auth/domain/dto/AuthResetPasswordDTO.java
package br.com.guizzo.projectbasic.modules.auth.domain.dto;


import br.com.guizzo.projectbasic.shared.annotations.Password;
import jakarta.validation.constraints.NotNull;
import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class AuthResetPasswordDTO{
        @NotNull(message = "Token inválido!")
        private String token;

        @Password
        private String password;
}

File: src/modules/auth/domain/dto/auth-reset-password.dto.ts
import { IsString } from 'class-validator';
import { IsPassword } from 'src/shared/decorators/is-password.decorator';

export class AuthResetPasswordDto {
  @IsPassword()
  password: string;

  @IsString({ message: 'Token inválido!' })
  token: string;
}
